rootfs/usr/local/bin/foremand-supervisor

```diff
@@ -159,6 +159,12 @@ elsif ARGV[0].downcase == "start"
 elsif ARGV[0].downcase == "run"
   start_server(ARGV[1] || "production")
 else
-  puts "Unknown command. Usage: formand start"
+  puts "Unknown command. Usage: formand <start|run>"
   exit 1
 end
+
+private
+
+def load_dotenv_if_present(app_env)
+  return "-e #{APP_ROOT}/.env.#{app_env}" if File.exist?("#{APP_ROOT}/.env.#{app_env}")
```

Comment:
That looks good, can you also add support for `.env.#{app_env}.local`?

Reply:
Added per suggestion, but I have changed the order to match https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use

.env.x.local takes precedence over .env.x